data/hai/unfettered hunger.txt

```diff
@@ -0,0 +1,395 @@
+					goto fight
+
+			label bribe
+			action
+				payment -500
+			`	You see movement in your peripheral vision just as you feel the credits pulled from your hand. One of the children off to your side darted forward before you could react and is now handing your credits to the leader. She checks the amount and then gives you a satisfied smile. At her nod, the group vanishes into the life of the street.`
+			`	You make your way quickly back to the spaceport, certain that the kids are shadowing you. It's with some relief that you make it through the security checkpoint.`
+				decline
+			
+			label fight
+			action
+				payment -5000
+			`	You reach for your sidearm only to find that it isn't there. It seems like a dozen pairs of paws are on you, some tearing at your clothes and some climbing on you. You spin around to try and shake them off. You end up rolling on the ground and then, as suddenly as it began, everything stops and you are alone in the street. You check yourself quickly and you seem to be uninjured. Your clothes are in tatters, though, and your sidearm, communicator, and bag are nowhwere to be seen.`
+			`	You make your way back to the spaceport. For once, it's useful to be the alien. The guards are quickly able to quickly clear you even without documentation. You hear them laughing as you head for the safety of your ship. You've no doubt that the story of he monkey robbed by children is already making the rounds.`
+				decline
+
+
+
+mission "Unfettered Hunger Aid"
+	name "Emergency aid shipment to Unfettered"
+	job
+	repeat
+	clearance "After speaking with the Unfettered and explaining your presence, they give you permission to land with the cargo."
+	description "Deliver <tons> of food to the Unfettered on <destination>. You will be paid <payment>."
+	cargo "food (aid)" 55 2 .05
+	to offer
+		has "unfettered: hunger"
+		random < 25
+		"cargo space" > 85
+		has "Ask the Hai about the Unfettered: offered"
+	source
+		government "Hai"
+	destination
+		distance 5 6
+		attributes "unfettered"
+	on accept
+		dialog phrase "unfettered aid pickup dialog"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		payment 10000 2800
+		"reputation: Hai" += 5
+		"reputation: Hai (Wormhole Access)" += 5
+		"reputation: Hai Merchant" += 5
+		"reputation: Hai Merchant (Human)" += 5
+		"reputation: Hai Merchant (Sympathizers)" += 5
+		"unfettered: hunger stops" += 100
+		dialog phrase "unfettered aid payment dialog"
+
+
+mission "Unfettered Hunger Tribute"
+	job
+	repeat
+	name "Significant Hai Tribute to <planet>"
+	description "Collect a tribute payment of <tons> of food from <stopovers>, and bring it to the Unfettered on <destination>. Your share of the tribute will be <payment>."
+	cargo "food (tribute)" 55 2 .05
+	source
+		attributes "unfettered"
+	stopover
+		distance 5 6
+		attributes "hai"
+		attributes "spaceport"
+	to offer
+		has "Unfettered Jump Drive 2: offered"
+		has "unfettered: hunger"
+        random < 40
```

Comment:
```suggestion
	random < 40
```
Use tabs instead of spaces.

Reply:
I thought I already fixed this but apparently not
I have a local PR to my main branch if you want to check the CI btw